.vsts.pipelines/builds/matrix.yml

```diff
@@ -41,9 +41,10 @@ jobs:
     reportPrebuiltLeaks: false
     systemLibunwind: false
     matrix:
-      Production: {}
+      Production: { generatePrebuiltBurndown: true, runUnitTests: true }
```

Comment:
Unit tests have been disabled in master until the entire product builds.  PR to re-enable: https://github.com/dotnet/source-build/issues/1549

Reply:
Ah, this snuck in because of the way I resolved a manual cherry pick. Undoing.